STARTING UP.............................
  Date and Time:    2024-05-01 at 07:59 PM
  Operating System: nt Windows 10
  Python Version:   3.11.4
  Path to Interpreter:  C:\Users\ksu09\AppData\Local\Programs\Python\Python311\python.exe
======================================================================

2024-05-01 19:59:08,913 - INFO - Called recreate_database().       
2024-05-01 19:59:08,913 - INFO - Called drop_table().
2024-05-01 19:59:08,924 - INFO - Table 'pets' dropped successfully.
2024-05-01 19:59:08,924 - INFO - Called create_table().
2024-05-01 19:59:08,933 - INFO - Table 'pets' created successfully.
2024-05-01 19:59:09,050 - INFO - Called process_one().
2024-05-01 19:59:09,050 - INFO -   Called insert_pet() with process=P1, name=Ace, breed=Dog.      
2024-05-01 19:59:09,050 - INFO - Called process_two().
2024-05-01 19:59:09,051 - INFO -   Called insert_pet() with process=P2, name=Cooper, breed=Rabbit.
2024-05-01 19:59:09,053 - INFO - Called process_three().
2024-05-01 19:59:09,053 - INFO -   Called insert_pet() with process=P3, name=Emma, breed=Rabbit.  
2024-05-01 19:59:12,061 - INFO -   Called insert_pet() with process=P2, name=Dingo, breed=Dog.
2024-05-01 19:59:14,663 - ERROR - ERROR while P3 inserting pet Emma: database is locked
2024-05-01 19:59:14,663 - INFO -   Called insert_pet() with process=P3, name=Felix, breed=Cat.
2024-05-01 19:59:14,679 - ERROR - ERROR while P1 inserting pet Ace: database is locked        
2024-05-01 19:59:14,679 - INFO -   Called insert_pet() with process=P1, name=Buddy, breed=Dog.